.copr/yggdrasil.spec.in

```diff
@@ -0,0 +1,80 @@
+%bcond_without check
+
+# https://github.com/redhatinsights/yggdrasil
+%global goipath         github.com/redhatinsights/yggdrasil
+%global commit          @COMMIT@
+
+%gometa -f -i
+
+# Manually redefine %%dist to work around an issue in COPR where the build root
+# that creates the srpm does not define a value for %%dist. This should *NOT* be
+# carried in downstream; this is strictly an upstream/COPR/CI workaround.
+%global dist %{distprefix}.fc%{fedora}
+
+%global common_description %{expand:
+yggdrasil is a system daemon that subscribes to topics on an MQTT broker and
+routes any data received on the topics to an appropriate child "worker" process,
+exchanging data with its worker processes through a D-Bus message broker.}
+
+%global golicenses      LICENSE
+%global godocs          CONTRIBUTING.md README.md
+
+Name:           yggdrasil
+Version:        0.3.1
+Release:        %autorelease -p
+Summary:        Remote data transmission and processing client
+
+License:        GPL-3.0-only
+URL:            %{gourl}
+Source:         %{gosource}
+
+BuildRequires:  systemd-rpm-macros
+BuildRequires:  meson
+BuildRequires:  pkgconfig(dbus-1)
+BuildRequires:  pkgconfig(systemd)
+BuildRequires:  pkgconfig(bash-completion)
+
+%description %{common_description}
+
+%gopkg
```

Comment:
Is `%gopkg` really necessary? Because there is e.g. `%description` in spec file.

Reply:
Yes. [`%gopkg`](https://pagure.io/go-rpm-macros/blob/master/f/rpm/macros.d/macros.go-srpm#_138) generates the subpackages automatically. I do not want to maintain that manually.